packages/infra/lib/api-stack.ts

```diff
@@ -667,6 +667,25 @@ export class APIStack extends Stack {
       bucket: medicalDocumentsBucket,
     });
 
+    // CONVERT API
+    const convertResource = api.root.addResource("convert");
+    const convertBaseResource = convertResource.addResource("v1");
+    const ccdaConvertResource = convertBaseResource.addResource("ccda");
+    const ccdaConvertBaseResource = ccdaConvertResource.addResource("to");
+    const ccdaToFhirConvertResource = ccdaConvertBaseResource.addResource("fhir");
+    const ccdaToFhirLambda = new lambda.DockerImageFunction(this, "convertApiCcdaToFhir", {
+      functionName: "convertApiCcdaToFhir",
+      vpc: this.vpc,
+      code: lambda.DockerImageCode.fromImageAsset("../fhir-converter", {
+        file: "Dockerfile.lambda",
+      }),
+      timeout: Duration.minutes(1),
+      memorySize: 1024,
+    });
+    ccdaToFhirConvertResource.addMethod("POST", new apig.LambdaIntegration(ccdaToFhirLambda), {
+      apiKeyRequired: true,
+    });
```

Comment:
Can we move this to a nested stack, please? We're too close to the limit, anything that can be on a nested stack should go to one IMO.